internal/sops/pgp/keysource_test.go

```diff
@@ -50,6 +50,23 @@ func TestNewGnuPGHome(t *testing.T) {
 	g.Expect(gnuPGHome.Validate()).ToNot(HaveOccurred())
 }
 
+func TestGnuPGHome_Import_With_Missing_Binary(t *testing.T) {
+	os.Setenv(SopsGpgExecEnv, "/does/not/exist")
```

Comment:
There is `t.Setenv` which prevents you from having to deal with the `os.Unsetenv` in cleanup.